de-lib/src/main/java/org/iplantc/de/theme/base/client/tools/ManageToolsViewDefaultAppearance.java

```diff
@@ -183,6 +183,16 @@ public String toolInfoError() {
         return toolMessages.toolInfoError();
     }
 
+    @Override
+    public String windowWidth() {
+        return "600";
+    }
+
+    @Override
+    public String windowHeight() {
+        return "500";
```

Comment:
The original dimensions of this window were 800x600.
Are we intentionally reducing it to 600x500 now?

Reply:
Yes I did change it intentionally. Felt it was too big but I can always change it back if others feels differently.

Reply:
I don't have an issue with the new size. I was just making sure it wasn't an untested accident ✅